src/net/sf/memoranda/ui/Wizard.java

```diff
@@ -0,0 +1,24 @@
+package net.sf.memoranda.ui;
+
+/**
+ * Classname: Wizard
+ * 
+ * Description: This is the Interface for PSP Wizard objects.
+ * 
+ * Version: 1.0
+ * 
+ * Date: 2/21/20156 
+ * 
+ */
+
+import java.awt.event.ActionEvent;
+
+public interface Wizard {
```

Comment:
While most examples illustrate the abstract factory pattern as requiring an interface, we should actually use an abstract class here, due to the fact that we're trying to abstract a concrete object rather than pure functionality. It will also help cut down a lot on the repeated code between the specialized types above, saving us maintenance headaches in the future.